core/src/chiplets/hasher.rs

```diff
@@ -62,26 +62,44 @@ pub const TRACE_WIDTH: usize = NUM_SELECTORS + STATE_WIDTH + 2;
 /// executing linear hash computation. These selectors can also be used for a simple 2-to-1 hash
 /// computation.
 pub const LINEAR_HASH: Selectors = [Felt::ONE, Felt::ZERO, Felt::ZERO];
+/// Unique id for the linear hash operation. Computed as 1 more than the binary composition of the
+/// chiplet and operation selectors [0, 1, 0, 0].
+pub const LINEAR_HASH_OP_ID: Felt = Felt::new(3);
 
 /// Specifies a start of Merkle path verification computation or absorption of a new path node
 /// into the hasher state.
 pub const MP_VERIFY: Selectors = [Felt::ONE, Felt::ZERO, Felt::ONE];
+/// Unique id for the merkle path verification operation. Computed as 1 more than the binary
+/// composition of the chiplet and operation selectors [0, 1, 0, 1].
+pub const MP_VERIFY_OP_ID: Felt = Felt::new(11);
```

Comment:
I originally did this, but I didn't like how disconnected they felt from the actual selector definitions. Not that we're planning to change the selectors, but it seemed like it would make it easier for them to become accidentally out of sync if there were to be a change. I could add a comment to the chiplets file, if you want (although of course that has the possibility of getting out of sync as well)

Reply:
Let's keep it as is then.